### PR TITLE
Counterstrike Soviet mission Top o' The World ported

### PR DESCRIPTION
**Pull request by AngryBirdz**
I've just finished porting this mission. Tell me if you spot something wrong ;)

Updated: I've fixed the rebase issues.

### CODE REVIEW

**Comment by pchote** [comment]
Looks like some rebase issues have crept into the commits here. You'll need to edit the files to remove the `<<<<<<< HEAD` through `>>>>>>> top-o-the-world` lines, and pick the right version of the code in between.

**Comment by AngryBirdz** [comment]
Fixed

File: mods/ra/maps/top-o-the-world/scu36ea.lua [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+	end)
+end
+
+Tick = function()
+
+-- Timer Setup
+	if RemainingTime == DateTime.Minutes(30) then
+		Media.PlaySpeechNotification(player, "ThirtyMinutesRemaining")
+	elseif RemainingTime == DateTime.Minutes(20) then
+		Media.PlaySpeechNotification(player, "TwentyMinutesRemainingg")
```
Comment (dragunoff) [inline mods/ra/maps/top-o-the-world/scu36ea.lua]: There's a typo - this should be `"TwentyMinutesRemaining"`

**Comment by pchote** [comment]
Can we get some testing and reviewing here?

**Review by abcdefg30 — COMMENTED** [review]
Looks good to me otherwise.

File: mods/ra/maps/top-o-the-world/scu36ea.lua [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+	end)
+
+	Trigger.AfterDelay(DateTime.Minutes(36), function()
+		if not ObjectiveTruck01.IsDead then ObjectiveTruck01.Kill() end
+	end)
+end
+
+Tick = function()
+
+-- Timer Setup
```
Comment (obrakmann) [inline mods/ra/maps/top-o-the-world/scu36ea.lua]: This would have been the perfect use case for my time limit PR. Oh well.

**Review by obrakmann — APPROVED** [review]
Mission looks good, :+1:

File: mods/ra/maps/top-o-the-world/scu36ea.lua (hunk 2 above)
Comment (pchote) [reply mods/ra/maps/top-o-the-world/scu36ea.lua]: Do you want to update this in your PR?

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/86b756a140b85d134aa3619a3a08db6d49f88b12)

File: mods/ra/maps/top-o-the-world/scu36ea.lua (hunk 2 above)
Comment (obrakmann) [reply mods/ra/maps/top-o-the-world/scu36ea.lua]: I could rebase it now and do it, yes.

**Pull request merged**